Add FBQS Integrator tool to diverse.yaml

MK-vet opened the pull request:
This PR adds the FBQS Integrator (fbqs_integrator) definition to diverse.yaml.  
FBQS Integrator merges quality-checked Fastp, Bracken and QUAST outputs into a unified STARAMR Excel report,  
including automatic QC flags (Passed/Warning/Fail) and separate sheets for Fastp, Bracken, Quast and overall quality metrics.

Contributors:
- Elena Lavinia Diaconu (elena.diaconu@izslt.it)
- Maciej Kochanowski (maciej.kochanowski@piwet.pulawy.pl)
- Sebastien Leclercq (sebastien.leclercq@inrae.fr)

Tool metadata:
- name: fbqs_integrator
- owner: SPAMR-VET Consortium (EUPAHW)

MK-vet reviewed (commented):
Thanks for your feedback. I’ve implemented the requested changes.

bgruening commented:
@MK-vet if you want to acknowledge your organisation and yourself you can add a <creator> tag to your tool. The toolshed owner is not meant for this and no user will see this. 
What other EU projects do, they contribute their tool to some public repo as Wolfgang mentioned and included a creator tag. This is then also nicely rendered in the Galaxy tool form.

MK-vet commented:
@wm75 @bgruening
Thanks for the feedback. I've made the requested changes:
- Fixed the owner field in the YAML file (changed to maciek as registered in toolshed)
- Added GPL-3.0 license to the tool
- Used the creator tag for organization info

wm75 commented:
The toolshed still says `This revision can be installed: False`

![image](https://github.com/user-attachments/assets/a6966d2f-62e3-4ed6-94fd-070723ed0e55)

When I run the xml through `planemo lint` it complains (correctly) that the `organization` tag isn't closed.

In general, it's highly recommended to use `planemo lint` and other [planemo](https://planemo.readthedocs.io/) commands to check the wrapper before uploading it to the toolshed.

MK-vet commented:
@wm75 The <organization name="SPAMR-VET Consortium"/> tag has been properly closed in fbqs_integrator.xml.

wm75 reviewed (approved):
Ok, lets give this a try then.
